packages/api/lib/writeRecords/write-granules.js

```diff
@@ -167,11 +167,14 @@ const getGranuleFromQueryResultOrLookup = async ({
  * Write a granule to PostgreSQL
  *
  * @param {Object} params
- * @param {Object} params.granuleRecord - An postgres granule records
+ * @param {Object} params.granuleRecord     - An postgres granule records
```

Comment:
Sorry I didn't catch this before, feel free to ignore this suggestion 🐜 
```suggestion
 * @param {Object} params.granuleRecord     - A postgres granule record
```